src/inverseRobot.js

```diff
@@ -23,8 +23,23 @@
  *
  * @return {object}
  */
+
 function inverseRobot(robot) {
-  // write code here
+  const newObj = {};
+  const values = Object.values(robot);
+  const keys = Object.keys(robot);
+
+  values.reverse();
+  keys.reverse();
+
+  for (let i = 0; i < values.length; i++) {
```

Comment:
try to use for in, that's let you avoid variables: values and keys, no need to use Object.values(robot);